code/modules/cooking_with_jane/step_types/use_tool.dm

```diff
@@ -42,5 +42,10 @@
 //Think about a way to make this more intuitive?
 /datum/cooking_with_jane/recipe_step/use_tool/calculate_quality(var/obj/added_item)
 	var/obj/item/tool/our_tool = added_item
-	var/raw_quality = (our_tool.get_tool_quality(tool_type) - tool_quality) * inherited_quality_modifier
+	var/raw_quality
+	if(usr.a_intent == I_HELP)
+		raw_quality = (our_tool.get_tool_quality(tool_type) - tool_quality) * inherited_quality_modifier
+	else
+		raw_quality = ((our_tool.get_tool_quality(tool_type) * -1) - tool_quality) * inherited_quality_modifier //Purposefully mucking up a recipe should invert the positive bonus into a negative
+		to_chat(usr, SPAN_NOTICE("You deliberately apply the [added_item] poorly, resulting in a worse dish."))
```

Comment:
Players might accidentally leave themselves off of help intent. This means this isn't always deliberate. For a better user experience, do not imply whether this was intentional, and instead try to imply their action intent matters.